civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -76,6 +76,17 @@
 * TEMP fixes till ui-select styling is fixed
 * Dropdown ul opacity was set to 0
 */
-.chr_leave-request-modal__form .ui-select-container ul {
-  opacity: 1 !important;
+.chr_leave-request-modal__form {
+ .ui-select-container ul {
```

Comment:
i will be consistent with rest of the file here and are we following that in test files where there is no newline for the first describe